Tests/MongoSwiftTests/.swiftlint.yml

```diff
@@ -0,0 +1,20 @@
+disabled_rules:
```

Comment:
what are the diffs from the top-level file and this?
it's too bad that it doesn't do something smart and apply *all* the .swiftlint.yml files as it goes down the tree, giving highest precendence to the lowest level. then we'd only have to include what's different in this file.

Reply:
Yeah, I thought the same thing. The diff is:
```diff
7a8,9
>   - force_try
>   - force_cast
19,23d20
< 
< excluded:
<   - build
<   - .build
<   - docs
```

In plain English: I've removed the `excluded:` and added ignores for `force_try` and `force_cast`.

P.S.: Did _not_ expect ````` ```diff````` to work but man, it looks great...

Reply:
😮 nice!